### DISCUSSION TITLE
Welp. It's the last day of summer. Nuxt 3 when?

### DISCUSSION THREAD

**Discussion by HenrijsS** [nuxt/nuxt#16810] [General]
![image](https://user-images.githubusercontent.com/49265222/187709798-b12e5d70-884f-441a-b9e5-25e81e2b1ec2.png)

Is it going to release today? 😆

**Comment by Saurou** [comment]
You were early, as last day of Summer was yesterday.

**Comment by bot08** [comment]
Soon))) As pi0 said, there will be 2-3 more RC versions. ([#7565](https://github.com/nuxt/framework/pull/7565) )

**Comment by zhaoweizhong** [comment]
How about the Auth module?
